### ISSUE TITLE
Poor performance on NeRF Blender Synthetic Data & potential bugs

### ISSUE THREAD

**Issue by dragonlong** [nerfstudio-project/nerfstudio#806]
**Describe the bug**
Hi, when I was trying to test nerfstudio-0.1.5 on the original NeRF dataset by using `./scripts/benchmarking/launch_train_blender.sh -m nerfacto -g 0`, it seems the training loss for some datasets will suddenly jump into NaN for some cases, or the losses will not converge well after 2k steps in my case. Another concern is that the testing performance doesn't seem to match with expected performance, the psnr should reach > 30dB after some training. Is there anything special that I need to take care of when doing this testing? I have attached the screenshots below to help diagnosis. Thanks!

**To Reproduce**
Use `nerfstudio-0.1.5`, and run `./scripts/benchmarking/launch_train_blender.sh -m nerfacto -g 0` on NeRF blender dataset

**Expected behavior**
The performance should at least match with the performance reported with previous NeRF papers, like >23dB for drums, and >30dB for others

**Screenshots**
<img width="1686" alt="image" src="https://user-images.githubusercontent.com/8425887/196750189-d6b516ea-3170-453a-90f4-56b796647f11.png">
<img width="1685" alt="image" src="https://user-images.githubusercontent.com/8425887/196751594-9129111e-9473-496a-8561-d048cd27f8f2.png">

**Comment by tancik** [comment]
It looks like you are running the `nerfacto` model. This model is designed for real world unbounded scenes, as such the configuration is not ideal for unbounded synthetic scenes like the blender dataset. For example the following flags should be set: `--pipeline.model.near-plane 2. --pipeline.model.far-plane 6. --pipeline.datamanager.camera-optimizer.mode off --pipeline.model.use-average-appearance-embedding False`.
These alone aren't sufficient, other methods hardcode into the model that the background is white which is not done in `nerfacto` and other method uniformly sample the space which is also not done in `nerfacto`.

**Comment by dragonlong** [comment]
Hi Matthew, 

Thanks for the feedback! I have launched again the experiments with both instant-ngp and nerfacto, now the training profile looks normal to me at least from the training side. Will close this ticket afterwards
![image](https://user-images.githubusercontent.com/8425887/197074712-15ccdfd2-5f63-4ce5-b17a-da809bdd419d.png)

**Comment by dubrovin-sudo** [comment]
> --pipeline.model.near-plane 2. --pipeline.model.far-plane 6. --pipeline.datamanager.camera-optimizer.mode off --pipeline.model.use-average-appearance-embedding False

This command give an error:
` Unrecognized or misplaced arguments: --pipeline.datamanager.camera-optimizer.mode`

**Comment by samadbarrikhojasteh** [comment]
Hi,
You need to put them before the `--data` and `--output-dir` like:

`ns-train nerfacto --vis viewer --pipeline.model.predict-normals True --pipeline.model.near-plane 2. --pipeline.model.far-plane 6. --pipeline.datamanager.camera-optimizer.mode off --pipeline.model.use-average-appearance-embedding False --data /.../output_colmap --output-dir /.../output_result nerfstudio-data`
